src/test/resources/gatling-debug/src/gatling/scala/GatlingDebugSimulation.scala

```diff
@@ -42,5 +42,5 @@ class GatlingDebugSimulation extends Simulation {
 
   println(s"@@@@.sys ${write(System.getProperties.asScala)}}")
 
-  setUp(scenario("Scenario Name").pause(1).inject(atOnceUsers(0)))
+  setUp(scenario("Scenario Name").pause(1).inject(atOnceUsers(1)))
```

Comment:
@eshepelyuk It looks like this causes Gatling 3.4.0 to hang. Not a blocking issue, behavior in this case is unspecified so far and should probably throw an Exception here. We'll fix in 3.4.1